ui/StatusQ/src/StatusQ/Components/private/statusMessage/StatusTextMessage.qml

```diff
@@ -63,7 +72,7 @@ Item {
 
         width: parent.width
         height: effectiveHeight + d.showMoreHeight / 2
-        opacity: opMask.active ? 0 : 1
+        visible: !showMoreOpacityMask.active && !horizontalOpacityMask.active
```

Comment:
This will again reintroduce the bug I just fixed: https://github.com/status-im/status-desktop/issues/8116

Reply:
Ah thank you for pointing that out. There was a conflict I've resolved, but I didn't realize that using `opacity` would be mandatory here.

So would be sticking to `opacity` be fine while keeping the expression in tact?

Reply:
Yup, just stick to `opacity` instead of `visible`

Reply:
Updated and tested the bug you've fixed in #8116 
This works as expected now.

Thanks again for pointing this out

Reply:
Perfect, thanks :hugs: